samples/todo-app/README.md

```diff
@@ -0,0 +1,113 @@
+- A statistics API exposes the cached statistics for the frontend to show.
+
+In all, this extended TODO application is composed of six inter-related components.
+
+## Prerequisites
+
+- Kubectl CLI (connected to a cluster)
+- Bridge to Kubernetes VS Code extension
+
+## Deploy the application
+
+In this example, we will use a local cluster, MiniKube.
+
+First, create a namespace for the sample.
+
+```
+kubectl create namespace todo-app
+```
+
+Then, apply the deployment manifest:
+
+```
+kubectl apply -n todo-app -f deployment.yaml
+```
+
+This is a simple deployment that exposes the frontend using a service of type `LoadBalancer`. Wait for all the pods to be running and for the external IP of the `frontend` service to become available.
+
+If you are testing with MiniKube, you will need to use `minikube tunnel` in  to resolve an external IP.
+
+```
+kubectl get services -n todo-app
+
+NAME          TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)        AGE
+frontend      LoadBalancer   10.0.49.177    52.149.235.179   80:30145/TCP   18h
+```
+
+Browse to the application using the external IP and give it a spin. As you add, complete and delete todos, notice that the stats page updates with the expected metrics
+
+## Debug the stats-api service
+
+We will now use the Bridge to Kubernetes extension to demonstarte how traffic from the Kubernetes cluster can be redirected to a locally running version of the stats-api. 
+
+```
+cd stats-api/
+```
+
+Open the source code for the stats-api in VS Code.
+
+```
+code .
+```
+
+Place a breakpoint on line 17 of server.js.
+
+Open the `Command Pallette (Ctrl + SHIFT + P) or (CMD + SHIFT + P)` and type Bridge to Kubernetes. Select the `"Bridge to Kubernetes: Configure"` option.
+
+![](/images/bridge_configure.png)
+
+You are prompted to configure the service you want to replace, the port to forward from your development computer, and the launch task to use.
+
+Choose the `stats-api` service. 
+![](/images/select_service.png)
+
+After you select your service, you are prompted to enter the TCP port for your local application. For this example, enter `3001`.
+![](/images/enter_port.png)
+
+Choose `Run Script: dev` as the launch task.
+![](/images/launch_task.png)
+
+> Note: You will be prompted to allow the EndpointManager to run elevated and modify your hosts file.
+
+You have the option of running isolated or not isolated. If you run isolated, only your requests are routed to your local process; other developers can use the cluster without being affected. If you don't run isolated, all traffic is redirected to your local process. For more information on this option, see [Using routing capabilities for developing in isolation](https://docs.microsoft.com/en-us/visualstudio/containers/overview-bridge-to-kubernetes?view=vs-2019#using-routing-capabilities-for-developing-in-isolation). For this example, we will proceed with non-isolated.
```

Comment:
It might be a good idea to have this doc show the isolated mode. That way we can talk about the nip.io URL and it doesn't come as a surprise to users who see it for the first time.

Reply:
or if we want, we can talk to both scenarios if that makes sense.